test/api/v01/vrp_test.rb

```diff
@@ -77,6 +77,15 @@ def test_ignore_unknown_parameters
     submit_vrp api_key: 'demo', vrp: vrp
   end
 
+  def test_refute_solver_and_solver_parameter
+    vrp = VRP.toy
+    vrp[:configuration][:resolution][:solver] = true
+    vrp[:configuration][:resolution][:solver_parameter] = 0
```

Comment:
Do we really want to reject this case ? These are not uncompatible.

Reply:
These parameters were never expected to be send together, `solver_parameter` is now deprecated for a long time and this way we have no ambiguity between these two fields ? 
What should we do with solver = true and solver_parameter = -1 ?

At this purpose we add a mutually_exclusive condition.